test/e2e/selectors/selectors_test.go

```diff
@@ -131,14 +147,14 @@ var _ = Describe("Field Selectors in Flight Control", Ordered, func() {
 	Context("Advanced Functionality Tests", func() {
 		It("Advanced Functionality Tests", Label("77947"), func() {
 			By("filters devices by multiple field selectors")
-			startTimestamp, _ := generateTimestamps()
-			out, err := harness.CLI("get", "devices", "-l", "region=eu-west-1", "--field-selector",
+			startTimestamp, _ := GetCurrentYearBounds()
+			out, err := harness.RunGetDevices("-l", "region=eu-west-1", "--field-selector",
```

Comment:
the region is used so many times, please make a var and get it from the device you created